File: CHUONG_5/5.08_GridView/app/src/main/java/com/example/gridview/data/model/Country.java
package com.example.gridview.data.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Country {
    @JsonProperty("name")
    private String nation;

    @JsonProperty("capital")
    private String capital;

    @JsonProperty("flag")
    private String flag;

    @JsonProperty("area")
    private int area;

    @JsonProperty("density")
    private int density;

    @JsonProperty("population")
    private float population;

    @JsonProperty("world_share")
    private String worldShare;

    public Country() {}

    public String getNation() {
        return nation;
    }

    public String getCapital() {
        return capital;
    }

    public String getFlag() {
        return flag;
    }

    public int getArea() {
        return area;
    }

    public float getPopulation() {
        return population;
    }

    public int getDensity() {
        return density;
    }

    public String getWorldShare() {
        return worldShare;
    }
}

File: CHUONG_10/10.2_ex1/app/src/main/java/com/example/myapplication/data/Country.kt
package com.example.myapplication.data

import com.fasterxml.jackson.annotation.JsonProperty

data class Country (
    @JsonProperty("name") val name: String,
    @JsonProperty("flag") val flag: String,
    @JsonProperty("capital") val capital: String,
    @JsonProperty("population") val population: Float,
    @JsonProperty("density") val density: Int,
    @JsonProperty("area") val area: Int,
    @JsonProperty("world_share") val worldShare: String
    )